### PR TITLE
[CLU] 1.1.0b2

### PR DESCRIPTION
**Pull request by kristapratico**
Done:
- Regenerates with GA DPG targeting 2022-05-15-preview
- Adds the authoring client APIs which we GA'd in 1.0.0
- Adds back @mshaban-msft's work on conv summarization and pii summarization for samples, tests, and docs 
- Adds AAD support for ConversationAnalysisClient and ConversationAuthoringClient in the beta 

Pending:
- ~Possible rename for begin_conversation_analysis to align with .NET~ (we'll revisit next beta)
- AAD doesn't work in 2022-05-15-preview (service team notified - tests are pinned to 2022-05-01)
- Authoring APIs throw 404 in 2022-05-15-preview / UK South (service team notified - tests are pinned to 2022-05-01)

### CODE REVIEW

**Review by iscai-msft — APPROVED** [review]
pretty much a ba but generated code looks good

File: sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md [hunk 1]
```diff
@@ -140,7 +140,16 @@ directive:
...
 ```
 
-```yaml $(tag) == 'release_runtime_1_0'
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Rename Runtime client async operation
+directive:
+    - from: swagger-document
+      where: $["paths"]["/analyze-conversations/jobs"]["post"]
+      transform: >
+          $["operationId"] = "ConversationAnalysis";
```
Comment (heaths) [inline sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md:149]: Interesting. You don't need to include the "method name" e.g. `ConversationAnalysis_Foo`?

File: sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md [hunk 2]
```diff
@@ -149,10 +158,50 @@ directive:
...
+# Remove async GET operation status
+directive:
+    - from: swagger-document
+      where: $["paths"]
+      transform: >
+          delete $["/analyze-conversations/jobs/{jobId}"];
+```
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Remove async cancel operation
```
Comment (heaths) [inline sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md:194]: Interesting. Why? Do you not support cancel, or is it built-in? That said, it shouldn't return an object like I did (and maybe you copied) for the inference client, but doesn't hurt. Just not a pattern we (the SDK teams) want to do going forward after talking with Johan et. al.

File: sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md (hunk 1 above)
Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md:149]: No - ConversationAnalysis _is_ the method name (for now) as this generates as `begin_conversation_analysis`. In Python DPG, if there is something before the `_` then all operations get put under an operation group. We don't want that or else user code will have to look something like this: 

```
client.operation_group.analyze_conversation(...)
```

File: sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md [hunk 3]
```diff
@@ -149,10 +158,50 @@ directive:
...
+      where: $["paths"]["/analyze-conversations/jobs"]["post"]
+      transform: >
+        $["parameters"][1]["x-ms-client-name"] = "task";
+```
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Give analyze job LRO a return type
+directive:
+  - from: swagger-document
+    where: '$.paths["/analyze-conversations/jobs"].post'
```
[the remark this answers is not in the document]
Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md:174]: yeah this was a lazy copy/paste

File: sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md (hunk 2 above)
Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md:194]: I posted in our chat yesterday that we're not ready to expose this quite yet:

> I spoke to Johan about cancel and he would like us to skip it for this release. DPG currently does not provide the poller metadata to cancel an operation and is something they will work on. Therefore the 1.1.0b2 for Python will only add the authoring APIs and regen with GA DPG.

**Pull request merged**